mappings/net/minecraft/entity/passive/TadpoleBrain.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_7111 net/minecraft/entity/passive/TadpoleBrain
 	METHOD method_41400 create (Lnet/minecraft/class_4095;)Lnet/minecraft/class_4095;
 		ARG 0 brain
-	METHOD method_41401 (Lnet/minecraft/class_7110;)V
+	METHOD method_41401 tadpoleActivityUpdate (Lnet/minecraft/class_7110;)V
```

Comment:
`updateActivities`? (note that AllayBrain's `resetIdleActivities` should be renamed as well)